airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -254,6 +254,7 @@ async def _wait_for_container_completion(self) -> TriggerEvent:
                         "message": "Container state failed",
                     }
                 )
+            self.log.debug("Container is not completed and still working.")
```

Comment:
nit:
            `self.log.debug("Container: %s has not completed and is still active.", <container name>)`